cpp/benchmarks/text/minhash.cpp

```diff
@@ -30,6 +30,7 @@ static void bench_minhash(nvbench::state& state)
   auto const row_width  = static_cast<cudf::size_type>(state.get_int64("row_width"));
   auto const hash_width = static_cast<cudf::size_type>(state.get_int64("hash_width"));
   auto const seed_count = static_cast<cudf::size_type>(state.get_int64("seed_count"));
+  auto const b64        = state.get_int64("htype") == 64;
```

Comment:
Can we rename `b64` (base 64?) and `htype` (hash type)? We have pretty readable names for all the other variables in this file.
```suggestion
  auto const b64        = state.get_int64("htype") == 64;
```